server/pulp/server/webservices/urls.py

```diff
@@ -88,8 +87,11 @@
     url(r'^v2/roles/(?P<role_id>[^/]+)/$', RoleResourceView.as_view(), name='role_resource'),
     url(r'^v2/roles/(?P<role_id>[^/]+)/users/$', RoleUsersView.as_view(), name='role_users'),
     url(r'^v2/roles/(?P<role_id>[^/]+)/users/(?P<login>[^/]+)/$', RoleUserView.as_view(), name='role_user'),
-    url(r'^v2/tasks/$', TaskCollectionView.as_view(), name='task_collection'),
-    url(r'^v2/tasks/(?P<task_id>[^/]+)/$', TaskResourceView.as_view(), name='task_resource'),
-    url(r'^v2/users/$', UsersView.as_view(), name='users'),
-    url(r'^v2/users/(?P<login>[^/]+)/$', UserResourceView.as_view(), name='user_resource')
+    url(r'^v2/tasks/$', tasks.TaskCollectionView.as_view(), name='task_collection'),
```

Comment:
is there a reason why you changed from TaskCollectionView.as_view() to tasks.TaskCollectionView.as_view() ?
Is not it more readable the first option?